manifests/pipecd/values.yaml

```diff
@@ -10,6 +10,15 @@ ingress:
     # kubernetes.io/ingress.allow-http: "false"
     # kubernetes.io/ingress.global-static-ip-name: pipecd
 
+# ServiceAccount
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
```

Comment:
@mugioka  Please change the default value to `false` to avoid adding an unexpected service account for existing users.

Reply:
I fixed it.